### DISCUSSION TITLE
Interesting tool for generating CLI GIFs / Integration Testing

### DISCUSSION THREAD

**Discussion by spenserblack** [o2sh/onefetch#842] [General]
Not sure if we want to use this or not, but just wanted to bring attention to [vhs](https://github.com/charmbracelet/vhs), which is basically a language to write a script (in a `.tape` file) to run CLI commands and handle their outputs. The 2 main uses are
1. Creating GIFs of usage.
2. [Integration testing](https://github.com/charmbracelet/vhs#continuous-integration).

**Comment by o2sh** [comment]
That's very useful, thanks for sharing @spenserblack 👍 

I like this part:

> Use the .txt or .ascii output to generate golden files. Store these files in a git repository to ensure there are no diffs between runs of the 
tape file.

We could use this for e2e (end to end) snapshot testing: info + formatting + ascii logo.
